support-frontend/assets/components/checkoutTopUpAmounts/checkoutTopUpAmounts.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react';
+import {
+	amountOption,
+	amountOptionAction,
+	amountOptionValue,
+	amountsAndNavigationContainer,
+	amountsContainer,
+	container,
+	hrCss,
+	standfirst,
+	supportTotalContainer,
+	title,
+} from './checkoutTopUpAmountsStyles';
+
+export interface CheckoutTopUpAmountsProps {
+	currencySymbol: string;
+	timePeriod: string;
+	amounts: number[];
+	isWithinThreshold: boolean;
+	handleAmountUpdate?: (updateAmountBy: number, index: number) => void;
+	customMargin?: string;
+}
+
+export function CheckoutTopUpAmounts({
+	currencySymbol,
+	timePeriod,
+	amounts,
+	isWithinThreshold,
+	handleAmountUpdate,
+	customMargin,
+}: CheckoutTopUpAmountsProps): JSX.Element {
+	const [selectedTopUpamount, setSelectedTopUpamount] = useState<number>(0);
+	const amountsLength = amounts.length;
+
+	return isWithinThreshold ? (
```

Comment:
I'd suggest something like the following rather than a ternary

```typescript
if (!isWithinThreshold) return <></>

return (
	<section>
	// blah blah
```

It's just a little bit easier to read and makes it clear immediately that if the condition is false we return nothing, rather than the reader having to scroll all the way to the end of the component to find that out.